sdk/aqueduct/check_artifact.py

```diff
@@ -33,14 +33,12 @@ class CheckArtifact(Artifact):
     """
 
     def __init__(
-        self,
-        api_client: APIClient,
-        dag: DAG,
-        artifact_id: uuid.UUID,
+        self, api_client: APIClient, dag: DAG, artifact_id: uuid.UUID, from_flow_run: bool
     ):
         self._api_client = api_client
         self._dag = dag
         self._artifact_id = artifact_id
+        self._from_flow_run = from_flow_run
```

Comment:
Can you write a comment here describing what this variable represents? Its not super clear from the name in what scenarios it is `_from_flow_run` and what scenarios it isn't.